Gruntfile.js

```diff
@@ -0,0 +1,29 @@
+/* jslint node: true */
+'use strict';
+
+var makeup = require( 'make-up' );
```

Comment:
If you version bump make-up you should be able to do away with your entire Gruntfile